repos/katello-repos.spec

```diff
@@ -17,6 +17,9 @@ BuildArch:      noarch
 BuildRequires: sed
 BuildRequires: ruby
 BuildRequires: python
+%if 0%{?fedora} > 23
+BuildRequires: rubygems
+%endif
```

Comment:
fedora 24 added a rubypick package (ruby version manager) that is required to be installed in order to run the ruby command. Rubypick by itself isn't sufficient and it requires rubygems to run (not sure why on that part).